CODEOWNERS

```diff
@@ -54,6 +54,7 @@
 /jbmc/src/jdiff/ @smowton @peterschrammel
 /src/cpp/ @kroening @tautschnig @peterschrammel
 /src/solvers/smt2 @kroening @martin-cs @tautschnig @peterschrammel @allredj @romainbrenguier
+/src/solvers/smt2_incremental @peterschrammel @diffblue/diffblue-opensource
```

Comment:
May I suggest some of the owners of `/src/solvers/smt2` as well.  @tautschnig does fantastic reviews and @romainbrenguier has the deepest understanding of the string solver which might well be one area you can really improve on.

Reply:
Is this a hint that you would like to be a code owner as well?

Reply:
I have checked with Romain and it is better to leave him off this list, as reviewing PRs on this piece of work would not currently be a priority for him.

Reply:
Your code; it's up to you who is a owner!  I would at least add yourself so that you can authorise (or not) controls that others make.

Reply:
I am already in the `@diffblue/diffblue-opensource` group. I think I'll expand it to the group's members so that this is less opaque.

Reply:
I have pushed the expansion of the group.